lib/runners/processor/checkstyle.rb

```diff
@@ -33,6 +35,13 @@ def setup
         return Results::Failure.new(guid: guid, message: exn.message)
       end
 
+      #if config_linter[:config]
+      #  trace_writer.message "The `config` option in `#{config.path_name}` is specified. The Sider's recommended ruleset is ignored."
+      #else
+      #   trace_writer.message "The `config` option in `#{config.path_name}` is not specified. Sider uses our recommended ruleset instead."
+      #end
+
+      warn_recommended_config_option_release(:config, "early in December 2020")
```

Comment:
```suggestion
      warn_recommended_config_option_release(:config, "in mid December 2020")
```

Shall we postpone the release until mid December?

Reply:
Fixed.